src/main/java/build/buf/protovalidate/CustomOverload.java

```diff
@@ -205,12 +204,34 @@ private static Overload celContains() {
           if (lhsType == TypeEnum.Bytes) {
             byte[] receiver = (byte[]) lhs.value();
             byte[] param = (byte[]) rhs.value();
-            return Types.boolOf(Bytes.indexOf(receiver, param) != -1);
+            return Types.boolOf(bytesContains(receiver, param));
           }
           return Err.noSuchOverload(lhs, OVERLOAD_CONTAINS, rhs);
         });
   }
 
+  static boolean bytesContains(byte[] arr, byte[] subArr) {
```

Comment:
This is the main thing we got from Guava. Immutable types are great but for our usage we didn't require them anywhere. We could always (when upgrading to Java 11) switch to similar types on the List/Map methods.